app/controllers/api/v1/courses/enrollments_controller.rb

```diff
@@ -8,10 +8,11 @@ class EnrollmentsController < ApplicationController
 
         def show
           if enrollment.present?
-            render :show, status: :ok
+            render json: enrollment, status: :ok
```

Comment:
What do you think about using?
```suggestion
            respond_with enrollment
```